### PR TITLE
Use anonymized errors in populateScratchBucketGcsPath

### PR DESCRIPTION
**Pull request by EricEdens**
Ensure that anonymized error is used as the first argument to `daisy.Errf`.

### CODE REVIEW

File: cli_tools/common/utils/param/populator_test.go [hunk 1]
```diff
@@ -267,6 +268,7 @@ func TestPopulator_DeleteResources_WhenScratchBucketInAnotherProject(t *testing.
 		deleteResult         error
 		deleteExpected       bool
 		expectedError        string
+		expectedAnonError    string
```
Comment (zoran15) [inline cli_tools/common/utils/param/populator_test.go:271]: What is "anon"?

**Comment by EricEdens** [comment]
Anon as in anonymous or anonymized. Updated to expectedAnonymizedError

>

File: cli_tools/common/utils/param/populator_test.go (hunk 1 above)
Comment (EricEdens) [reply cli_tools/common/utils/param/populator_test.go:271]: Anon as in anonymous or anonymized. Updated to expectedAnonymizedError

Comment (zoran15) [reply cli_tools/common/utils/param/populator_test.go:271]: I still see `anon` in tests and in param_utils.go

Comment (EricEdens) [reply cli_tools/common/utils/param/populator_test.go]: ahh, good find. Updated all.

**Review by zoran15 — APPROVED** [review]

**Pull request merged**